nix/avr/simavr-32kb-ram.patch

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) The mlkem-native project authors
+# SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+--- a/simavr/cores/avr/iom128rfr2.h
++++ b/simavr/cores/avr/iom128rfr2.h
+@@ -6300,8 +6300,8 @@
+ 
+ #define SPM_PAGESIZE                    (256)
+ #define RAMSTART                        (0x200)
+-#define RAMSIZE                         (0x4000)
+-#define RAMEND                          (0x41FF)
++#define RAMSIZE                         (0x8000)
++#define RAMEND                          (0x81FF)
```

Comment:
Have you tried increasing this more? If we can go to 192 KB / 256 KB, we could easily do the same for mldsa-native.